### ISSUE TITLE
Text fails WCAG accessibility

### ISSUE THREAD

**Issue by vanessali824** [vanessali824/fun-with-git#1]
Royal blue(#4169e1) with a foreground of black fails using https://webaim.org/resources/contrastchecker/
